### PR TITLE
feat: /solutions/open-ran build

### PR DESCRIPTION
**Pull request by mtruj013**
## Done

- Built new page as per [doc](https://docs.google.com/document/d/1rqBh8oVNeDk-fg1jg-7-SErLgmi0naEzP4r50JhJUb4/edit?tab=t.0) and [design](https://www.figma.com/design/brgzRViEncgKKD4KzVnyXy/24.10-Canonical.com%2Fsolutions?node-id=703-7693&node-type=instance&m=dev)
- Drive by:
  - Updated spacing on /solutions/telco hero per UX request  

## QA

- View the site locally in your web browser at: https://canonical-com-1400.demos.haus/solutions/telco/open-ran
- Check against doc and design
- https://canonical-com-1400.demos.haus/solutions/telco and see that the hero spacing has been updated

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-14843

### CODE REVIEW

**Comment by eliman11** [comment]
Looks great, thank you! A few comments from me: 
- [ ] Could you please add alt text for the Intel logo?
- [ ] Missing space between 'Canonical Kubernetes' and 'offers the perfect..' in the first sentence in the section below the logo carousel 
- [ ] Link under 'Data science and ML platform for Open RAN' in the 'Open source data and AI/ML platforms for Open RAN'
 section should be: [Discover Charmed Kubeflow >](https://charmed-kubeflow.io/)
- [ ] I double checked this with Juan and it seems like the navigation to the page is missing. It should be a tab at the end in the telco bubble nav similar to 5G and 5G Edge
- [ ] Also unrelated to this page but noticed the spacing at the top of the hero in the [main telco page](https://canonical.com/solutions/telco) is missing. Would it be possible to add it in? Thank you!

**Comment by mattea-turic** [comment]
The image here has too much whitespace, so I customised the aspect ratio and added the new asset [here](https://assets.ubuntu.com/v1/0ab657ca-chart-1.png)
Screenshot 2024-10-17 at 17 23 44

And the one below is missing the 3% black background, so if you could apply that and I uploaded an image for this too, [same server](https://assets.ubuntu.com/manager?tag=telco-openran-assets)

Missing the 24px padding in these two places under the h2s
Screenshot 2024-10-17 at 17 40 16

and this one:
Screenshot 2024-10-17 at 17 41 45

there are two bottom margins being used here between the first and second set of links, which makes it to be 48px rather than 24px, so if you could remove one of those pls
Screenshot 2024-10-17 at 17 44 06

also, just noticed the hpe logo is cut off (problem with the img itself), so I'll upload new one to the same server as mentioned above

**Comment by mtruj013** [comment]
Thanks @mattea-turic and @eliman11! I've pushed the changes if you both could take another look? Mattea could I also ask you to implement one more update to the design? The bottom spacing for the "Learn more" section was taken from the original version of the mock-up and it's in the current version as well, could you update? The one I ended up removing the `p-section--shallow` from the "Reading" row. Thank you!
Screenshot from 2024-10-18 10-09-43
Screenshot from 2024-10-18 10-13-23

**Comment by eliman11** [comment]
Looks good to me, thank you!

File: templates/solutions/telco/open-ran.html [hunk 1]
```diff
@@ -0,0 +1,525 @@
...
+    <section class="p-strip is-deep">
+      <div class="u-fixed-width">
+        <div class="p-section--shallow">
+          <h2>
+            Canonical provides a full stack for your telecom infrastructure.
+            <br />
+            Get in touch today for your telecom needs.
+          </h2>
+        </div>
+        <div class="p-cta-block">
```
Comment (lizzochek) [inline templates/solutions/telco/open-ran.html:517]: There is no hr on the design here. Pls confirm if it should be there

**Comment by lizzochek** [comment]
Looks like there is a little too much space here compared to the design. Again pls confirm with he designers
Screenshot 2024-10-18 at 13 08 53

File: templates/solutions/telco/index.html [hunk 2]
```diff
@@ -16,32 +16,30 @@
...
+        <p>
+          Canonical is the trusted and agile partner you need to boost network performance, optimise costs and achieve digital transformation with open source.
+        </p>
+        <p>Get enterprise-grade security and support to innovate with confidence.</p>
+        <hr class="p-rule--muted" />
+        <a class="p-button--positive"
+            href="https://ubuntu.com/telco#get-in-touch">Contact us</a>
       </div>
     </div>
+    <div class="u-fixed-width u-align--center">
```
Comment (lizzochek) [inline templates/solutions/telco/index.html:34]: I believe this should be within p-image-section

**Review by lizzochek — APPROVED** [review]

File: templates/solutions/telco/index.html (hunk 2 above)
Comment (mtruj013) [reply templates/solutions/telco/index.html:34]: I don't think this is needed. From the [docs](https://vanillaframework.io/docs/patterns/images), the image container is meant to "Enhance images by distinguishing them from their background or controlling their container's aspect ratio." This iso isn't supposed to be on a different background color and the aspect ratio is already correct as is

**Pull request merged**